docker/dockerfiles/Dockerfile.worker

```diff
@@ -37,6 +37,7 @@ RUN python3.6 -m pip install --user --upgrade pip; \
 # Install code
 COPY codalab/lib codalab/lib
 COPY codalab/worker codalab/worker
+COPY codalab/common.py codalab/common.py
```

Comment:
I'm confused - how was this able to work before?  Since we don't copy other things like `server` or `worker_manager` either...

Reply:
@percyliang I just double-checked our codebase, it seems that we only use `codalab.lib` in the worker side code. That's why it was working before?

Reply:
Ah right, this is the worker code.  Makes sense.